README.md

```diff
@@ -91,18 +91,33 @@ Open JupyterLab: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybind
 
 ## Development
 
+First, set up the environment:
+
 ```shell
-mamba create -c conda-forge -y -n gator python jupyterlab=3
+mamba create -c conda-forge -y -n gator python jupyterlab
 mamba install -c conda-forge -y -n gator --file requirements_dev.txt
 conda activate gator
-pip install -e .
-jupyter server extension enable mamba_gator --sys-prefix
+```
 
+Clone the repo, install it with `pip`, and then watch for changes to the
+javascript:
+
+```bash
+git clone git@github.com:quansight/gator
+cd gator
+pip install -e .
 yarn install
-yarn run build:dev
-jupyter labextension link packages/common/ packages/labextension/
+./node_modules/.bin/lerna run watch --parallel  # <-- enable automatic rebuilds
```

Comment:
If you add a `watch` script to the root `package.json` running `lerna run watch --parallel`, then this can be just `yarn run watch`. It's generally preferable to include commonly used commands as scripts there, for discoverability and so you don't have to type in the relative path to the `lerna` bin.

Reply:
Thanks, I've added an appropriate script to `package.json`.